pwndbg/commands/context.py

```diff
@@ -82,14 +83,22 @@ def output():
         return open(str( config_output ), "w")
 
 # @pwndbg.events.stop
-@pwndbg.commands.Command
+
+parser = argparse.ArgumentParser()
+parser.description = "Print out the current register, instruction, and stack context."
+parser.add_argument("subcontext", nargs="*", type=str, default=None, help="Submenu to display: 'reg', 'disasm', 'code', 'stack', 'backtrace', and/or 'args'")
+@pwndbg.commands.ArgparsedCommand(parser)
 @pwndbg.commands.OnlyWhenRunning
-def context(*args):
+def context(subcontext=None):
     """
     Print out the current register, instruction, and stack context.
 
     Accepts subcommands 'reg', 'disasm', 'code', 'stack', 'backtrace', and 'args'.
     """
+    if subcontext is None:
+        subcontext = []
+    args = subcontext
```

Comment:
Or just:
```python
    args = subcontext or config_context_sections.split()
```
and we could also remove the latter  `if len(args) == 0` and some code above